tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/Im2txtRestDockerfile

```diff
@@ -18,35 +18,41 @@
 FROM inception-rest-tika
 MAINTAINER Apache Tika Team
 
-#Install python pillow. TODO: Remove this with the fix for TIKA 2398
+# Install python pillow. TODO: Remove this with the fix for TIKA 2398
 RUN pip install pillow
 
 # Download the pretrained im2txt checkpoint
 WORKDIR /usr/share/apache-tika/models/dl/image/caption/
 
 RUN \
-    wget https://www.dropbox.com/s/l9ignjpjk774n2z/model_main_untuned.zip?dl=0 \
-        -O model_main_untuned.zip && \
-        unzip model_main_untuned.zip && \
-        rm -rf model_main_untuned.zip && \
+    # To get rid of early EOF error
+    git config --global http.postBuffer 1048576000 && \
+    echo "We're downloading the checkpoint file for image captioning, the shell might look unresponsive. Please be patient."  && \
+    git clone https://github.com/USCDataScience/img2text.git && \
+    # Join the parts
+    cat img2text/models/1M_iters_ckpt_parts_a* >1M_iters_ckpt.tar.gz && \
+    tar -xzvf 1M_iters_ckpt.tar.gz && \
+    # Delete all files except 1M_iters_ckpt
+    rm -rf {1M_iters_ckpt.tar.gz,img2text}
 
+RUN \
     wget https://raw.githubusercontent.com/ThejanW/tika/master/tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/model_info.xml \
```

Comment:
please update this wget line to point to apache/master, then I think we're good to test/go @ThejanW